docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/ZAP.xml

```diff
@@ -0,0 +1,140 @@
+      logos, and services may be trademarks or registered trademarks of their
+      respective companies. All names and example data used in this manual are
+      fictitious. Any similarity to actual persons, living or dead, is purely
+      coincidental.</para>
+
+      <para />
+    </legalnotice>
+
+    <xi:include href="../../common/Version.xml" xpointer="FooterInfo"
+                xmlns:xi="http://www.w3.org/2001/XInclude">
+      <xi:fallback>
+        <para>
+          <emphasis>FooterInfo Failed to load</emphasis>
+        </para>
+      </xi:fallback>
+    </xi:include>
+
+    <xi:include href="../../common/Version.xml" xpointer="DateVer"
+                xmlns:xi="http://www.w3.org/2001/XInclude">
+      <xi:fallback>
+        <para>
+          <emphasis>DateVer Failed to load</emphasis>
+        </para>
+      </xi:fallback>
+    </xi:include>
+
+    <corpname>HPCC Systems<superscript>®</superscript></corpname>
+
+    <xi:include href="../../common/Version.xml" xpointer="Copyright"
+                xmlns:xi="http://www.w3.org/2001/XInclude">
+      <xi:fallback>
+        <para>
+          <emphasis>Copyright Failed to load</emphasis>
+        </para>
+      </xi:fallback>
+    </xi:include>
+
+    <mediaobject role="logo">
+      <imageobject>
+        <imagedata fileref="../../images/LN_Rightjustified.jpg" />
+      </imageobject>
+    </mediaobject>
+  </bookinfo>
+
+        <sect2 id="ZAP">
+          <title>Z.A.P. Utility</title>
+
+          <para>The Zipped Analysis Package (Z.A.P.) button is a utility for
+          collecting system information and encapsulating it into a shareable
+          package. It is a useful tool for reporting errors, inconsistencies,
+          or other unexpected behavior. When there is such an occurrence, this
+          utility packages up information to send for further analysis.</para>
+
+          <figure>
+            <title>Z.A.P. Button</title>
+
+            <mediaobject>
+              <imageobject>
+                <imagedata fileref="../../images/ECLWAZ00.jpg" />
+              </imageobject>
+            </mediaobject>
+          </figure>
+
+          <para>To use the Z.A.P. utility, press the Z.A.P. button on the
+          workunit details page from the appropriate workunit. The button
+          opens the Zipped Analysis Package dialog.</para>
+
+          <figure>
+            <title>The Zipped Analysis Package Dialog</title>
+
+            <mediaobject>
+              <imageobject>
+                <imagedata fileref="../../images/ECLWAZ01.jpg" />
+              </imageobject>
+            </mediaobject>
+          </figure>
+
+          <para>Some of the required fields are populated. Fill in the
+          corresponding values under Description, History, and Timings fields.
+          Optionally, you can password protect the ZAP package and choose to
+          include slave logs. Slave logs are not included by default. If there
```

Comment:
Should not use the deprecated term. I believe the interface names have been corrected to avoid offensive terms.